tests/scripts/t3000/run_t3000_unit_tests.sh

```diff
@@ -134,6 +134,24 @@ run_t3000_grok_tests() {
   fi
 }
 
+run_t3000_unet_shallow_tests() {
```

Comment:
new functions introduced in this script will also need to add to 

`.github/workflows/t3000-unit-tests.yaml`

append code below.
```
{ name: "t3k unet tests", arch: wormhole_b0, cmd: run_t3000_unet_shallow_tests, timeout: 30, owner_id: U06ECNVR0EN}, #Eyon Land
```

This is to ensure that these tests are actually run in CI as CI individually calls functions from shell scripts

Reply:
Gotcha - fixed!